services/catarse/app/models/contribution.rb

```diff
@@ -24,6 +24,7 @@ class Contribution < ApplicationRecord
   accepts_nested_attributes_for :address, allow_destroy: true, limit: 1 #payment address
   accepts_nested_attributes_for :address_answer, allow_destroy: true #survey answer addresses
   after_save :index_on_common
+  before_save :updated_service_slip_fee
```

Comment:
**Sugestão: **
Penso que faria mais sentido que esse campo `service_slip_fee` do projeto fosse preenchido no momento da criação do projeto, ao invés de ser um `before_save` na contribution.

**Dúvida: **
Desse modo que está sendo feito, um projeto em andamento terá sua taxa alterada para a nova taxa quando depois do deploy dessa atividade um boleto for criado, e nos dados do projeto ficará: Taxa de boleto: R$ 2,00. Até aí tudo bem, mas aí terão outros boletos que já foram gerados e pagos pra esse projeto antes do deploy que não terão a taxa. É pra ser assim mesmo?

Reply:
Falei com o ton, e vai ser feito um levantamento com o thiago. Talvez será gerado um flag  pra só valer pra projetos que tao entrando no ar a partir do dia X.

Reply:
Nem precisa de flag @gabrielras, só colocar no after create do project, e depois rodar uma rotina pra atualizar as taxas do que estão em rascunho.

Reply:
Excelente pescada @stephannv 

Falha minha não ter levantado essa hipótese do flag / after create project (pois imaginava que seria algo bem complicado). Mas vamos então com essa solução de usar o after create project e fazemos então uma rotina, logo após que o deploy for feito, para atualizar os projetos que estão em rascunho. @stephannv @gabrielras

Reply:
Boa @thiagocatarse